SPECS/librelp/librelp.spec

```diff
@@ -48,6 +48,11 @@ make DESTDIR=%{buildroot} install
 #and since tests are not using any valgrind functionality,
 #disabling valgrind.
 sed -ie 's/export valgrind=.*/export valgrind""/' tests/test-framework.sh
+
+# The tls-basic-brokencert test is marked unstable in upstream source, so disable it.
```

Comment:
Thanks for adding the comments explaining what's wrong and why we went this route.